website/versioned_docs/version-v1.2/cli.md

```diff
@@ -27,28 +27,28 @@ pnpm run hardhat
+| `deployPoll`         | Deploy a new poll                                                  | `-vk, --vkRegistryAddress <vkRegistryAddress>`: The vk registry contract address <br/> `-t, --duration <pollDuration>`: The poll duration <br/> `-i, --int-state-tree-depth <intStateTreeDepth>`: The int state tree depth <br/> `-b, --msg-batch-depth <messageTreeSubDepth>`: The message tree sub depth <br/> `-m, --msg-tree-depth <messageTreeDepth>`: The message tree depth <br/> `-v, --vote-option-tree-depth <voteOptionTreeDepth>`: The vote option tree depth <br/> `-pk, --pubkey <coordinatorPubkey>`: The coordinator public key <br/> `-uq, --use-quadratic-voting"`: Whether to use quadratic voting <br/> `-x, --maci-address <maciAddress>`: The MACI contract address <br/> `-q, --quiet`: Whether to print values to the console <br/> `-r, --rpc-provider <provider>`: The rpc provider URL                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                     |
```

Comment:
there's no `-uq` flag in v1.2?

missing this 
`-se, --subsidy-enabled <subsidyEnabled>` whether to deploy subsidy contract